tests/test_platform.py

```diff
@@ -218,6 +217,7 @@ def test_platform_execute_multiple_readout_pulses(qpu_platform):
     platform.execute_pulse_sequence(sequence, ExecutionParameters(nshots=nshots))
 
 
+@pytest.mark.skip(reason="no way of currently testing this")
```

Comment:
I believe we could completely drop this test, given that we removed the probability calculation.

Reply:
As there is already a PR in qibo about adding the probabilities to `MeasurementOutcomes`, I would keep this skipped, and, upon merge, reintegrate them. Unless they are not useful in general.

Reply:
The probabilities are useful to have. For this test, we could keep it as you say, but then we would need to update it to create `MeasurementOutcomes` instead of `CircuitResult` and use the `.probabilities` method instead of `backend.circuit_result_probabilities` that we are removing.